
Training fold 1 for efficientnet_b0
Training fold 2 for efficientnet_b0
Training fold 3 for efficientnet_b0
Training fold 4 for efficientnet_b0
Training fold 5 for efficientnet_b0
Test accuracy: 60.000
Confusion Matrix:
 [[7 1 0 0 0 0 0 0 0 0 0]
 [3 5 0 0 0 0 0 0 0 0 0]
 [1 0 1 0 0 0 0 0 0 0 0]
 [1 0 0 1 0 0 0 0 0 2 0]
 [0 0 0 3 2 0 0 0 0 0 0]
 [0 0 0 0 0 0 0 1 0 0 0]
 [0 0 0 0 0 0 1 0 0 2 0]
 [0 0 0 0 0 0 0 6 1 0 0]
 [0 0 0 0 0 0 0 0 3 0 0]
 [0 3 0 0 1 2 1 0 0 9 0]
 [0 0 0 0 0 0 0 1 1 0 1]]
Test Precision: 63.452279
Test Recall: 60.000000
Test F1 Score: 59.367084
Matthews Correlation Coefficient: 54.048029
Traceback (most recent call last):
  File "/home/edramos/Documents/MLOPS/ImageClassification-MFG/main.py", line 375, in <module>
    test_model(model, test_loader, architecture, optimizer, scheduler, batch_size, image_size)
  File "/home/edramos/Documents/MLOPS/ImageClassification-MFG/main.py", line 245, in test_model
    wandb.log({"ROC Curve": roc_fig})
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 371, in wrapper
    return func(self, *args, **kwargs)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 334, in wrapper
    return func(self, *args, **kwargs)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1713, in log
    self._log(data=data, step=step, commit=commit)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1495, in _log
    self._partial_history_callback(data, step, commit)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1365, in _partial_history_callback
    self._backend.interface.publish_partial_history(
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 568, in publish_partial_history
    data = history_dict_to_json(run, data, step=user_step, ignore_copy_err=True)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/data_types/utils.py", line 52, in history_dict_to_json
    payload[key] = val_to_json(
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/data_types/utils.py", line 80, in val_to_json
    val = Plotly.make_plot_media(val)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/sdk/data_types/plotly.py", line 49, in make_plot_media
    val = util.matplotlib_to_plotly(val)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/wandb/util.py", line 637, in matplotlib_to_plotly
    return tools.mpl_to_plotly(obj)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/tools.py", line 111, in mpl_to_plotly
    matplotlylib.Exporter(renderer).run(fig)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/matplotlylib/mplexporter/exporter.py", line 53, in run
    self.crawl_fig(fig)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/matplotlylib/mplexporter/exporter.py", line 124, in crawl_fig
    self.crawl_ax(ax)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/matplotlylib/mplexporter/exporter.py", line 130, in crawl_ax
    self.draw_line(ax, line)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/matplotlylib/mplexporter/exporter.py", line 205, in draw_line
    self.renderer.draw_marked_line(
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/matplotlylib/renderer.py", line 442, in draw_marked_line
    line = go.scatter.Line(
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/graph_objs/scatter/_line.py", line 378, in __init__
    self["dash"] = _v
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/basedatatypes.py", line 4851, in __setitem__
    self._set_prop(prop, value)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/basedatatypes.py", line 5195, in _set_prop
    raise err
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/plotly/basedatatypes.py", line 5190, in _set_prop
    val = validator.validate_coerce(val)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/_plotly_utils/basevalidators.py", line 611, in validate_coerce
    self.raise_invalid_val(v)
  File "/home/edramos/miniconda3/envs/tf/lib/python3.9/site-packages/_plotly_utils/basevalidators.py", line 288, in raise_invalid_val
    raise ValueError(
ValueError:
    Invalid value of type 'builtins.str' received for the 'dash' property of scatter.line
        Received value: 'circle'
    The 'dash' property is an enumeration that may be specified as:
      - One of the following dash styles:
            ['solid', 'dot', 'dash', 'longdash', 'dashdot', 'longdashdot']
      - A string containing a dash length list in pixels or percentages
            (e.g. '5px 10px 2px 2px', '5, 10, 2, 2', '10% 20% 40%', etc.)